rsfbclient-diesel/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 //! The Firebird Diesel
 //!
-//! This crate re-export all diesel API with the intention of simulate the others [Diesel](https://docs.diesel.rs/diesel/index.html) implementations.
+//! This crate re-export all diesel API with the intention of simulate the others [Diesel](https://docs.diesel.rs/2.0.x/diesel/index.html) implementations.
```

Comment:
As diesel maintainer I would prefer not to reexport diesel here. Instead only the custom connection/backend implementation should be provided + users should just depend on diesel directly. That gives them much greater control about enabled features + makes it clearer where stuff is coming from.

Reply:
I will check this before finish the PR